tests/test_mask.py

```diff
@@ -35,26 +44,70 @@ def assertMask(self, mask: Mask):
             assert 0 <= property < 1, f"{propertyName} is out of range [0,1]"
 
     @staticmethod
-    def assertMaskReply(maskDict: dict):
+    def assertMaskPropertyReply(replyMask: Mask, expectedProperty: str, lowerThreshold: float = 0.9):
         """
-        Validate mask reply
+        Function check estimate property from reply
```

Comment:
may be `Function checks predominant property from reply`?

Reply:
+

Reply:
checks